code/model/DMSDocumentSet.php

```diff
@@ -216,6 +215,15 @@ public function addQueryFields($fields)
             if ($field instanceof ListboxField) {
                 $map = ($field->getName() === 'Tags__ID') ? $doc->getAllTagsMap() : $membersMap;
                 $field->setMultiple(true)->setSource($map);
+
+                if ($field->getName() === 'Tags__ID') {
+                    $field->setRightTitle(
+                        _t(
+                            'DMSDocumentSet.TAGS_RIGHT_TITLE',
+                            'Tags can be set in the taxonomies model admin'
```

Comment:
I think we should say "in the taxonomy area" which would be more CMS user friendly - model admin is a dev term.

Also could you add something like ", and can be assigned when editing a document"